frontend/src/Liked/Liked.js

```diff
@@ -0,0 +1,87 @@
+import { useReduce } from "../Reducer-context";
+import { getSortedData, getFilteredData } from "../App";
+import Navigator from "../Navigator";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+import { Navbar } from "../Navbar";
+
+export default function Liked() {
+  useEffect(() => {
+    dispatch({ type: "CLEAR_FILTER" });
+  }, []);
+  let { likedlist, dispatch, sortBy, showCategory } = useReduce();
+  let sortedData = getSortedData(likedlist, sortBy);
+  console.log("sorted", sortedData);
+
+  let filteredData = getFilteredData(sortedData, { showCategory });
+  console.log("filtered", filteredData);
+
+  function Likedlist() {
+    return filteredData
+      .filter((item) => item.islike !== false)
+      .map((item) => {
+        return (
```

Comment:
Add a name or indicator at the top of the page if Videos are there to show the user he's in Liked videos